### DISCUSSION TITLE
Need help with steps

### DISCUSSION THREAD

**Discussion by NikarashiHatsu** [saadeghi/daisyui#1984] [Q&A]
Is there's a way to swap the `step`'s `::before` and `::after`?

I mean, if I use `auto-rows-min` inside the `steps steps-vertical` class, the vertical bar on the left is not at full height.

<img width="954" alt="Screenshot 2023-06-12 at 11 26 23" src="https://github.com/saadeghi/daisyui/assets/43662951/e62634a3-9e98-4148-ad6a-8abc67a0c928">

Or, if anything I could to to fix the element, please let me know.

**Comment by saadeghi** [comment]
I will work on this today to see if I can find a way...

**Reply by NikarashiHatsu** [reply]
Thank you, I'm providing a Tailwind Play link just in case you need the reproduction code.
Sorry for no image.

[https://play.tailwindcss.com/ccefoAzrlu](https://play.tailwindcss.com/ccefoAzrlu)

**Reply by saadeghi** [reply]
I couldn't find a solution with current markup. And changing the markup would be a breaking change.

I will investigate more to see if I can find a new solution.

**Reply by NikarashiHatsu** [reply]
Do you have any temporary workaround to this issue?

**Reply by saadeghi** [reply]
Here's a workaround:  
https://play.tailwindcss.com/NdGfQhcLgj?file=css

I will add a `timeline` class next week that would fit this use case. I can't apply these changes to `steps` now because it will break some websites but we need something like timeline anyway and it fits better when we have items with different sizes.

**Reply by NikarashiHatsu** [reply]
Thank you!
